### PR TITLE
add first entry on packager

### PR DESCRIPTION
**Pull request by johanvos**

### CODE REVIEW

File: content/javafx/packaging/_index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+--name ${APP_NAME} \
+--main-class ${LAUNCHER_CLASS} \
+--main-jar ${MAIN_JAR} \
+--java-options -Xmx2048m \
+--runtime-image target/java-runtime \
+--icon ${ICON_PATH} \
+--app-version ${APP_VERSION} \
+--vendor "Gluon" \
+--copyright "Copyright © 2021 Gluon" \
+"$@"
```
Comment (abhinayagarwal) [inline content/javafx/packaging/_index.md:66]: This is most probably not required. We can pass `--type` by explicitly specifying a platform type or `${INSTALLER_TYPE}`

File: content/javafx/packaging/_index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  --multi-release ${JAVA_VERSION} \
+  --ignore-missing-deps \
+  --print-module-deps \
+  --class-path "target/installer/input/libs/*" \
+    target/classes/${MAIN_CLASS}.class`
+```
+
+Apart from those detected modules, you may specify additional modules, e.g.
+```
+manual_modules=,jdk.crypto.ec,jdk.localedata
```
Comment (abhinayagarwal) [inline content/javafx/packaging/_index.md:33]: Remove `,` from here and add it to L44 instead?

File: content/javafx/packaging/_index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  --compress=2  \
+  --strip-debug \
+  --add-modules "${detected_modules}${manual_modules}" \
+  --include-locales=en \
+  --output target/java-runtime
+```
+
+## Packaging
+
+The following command creates a native installer:
```
Comment (abhinayagarwal) [inline content/javafx/packaging/_index.md:51]: `s/native installer/JVM installer`?

File: content/javafx/packaging/_index.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+--main-class ${LAUNCHER_CLASS} \
+--main-jar ${MAIN_JAR} \
+--java-options -Xmx2048m \
+--runtime-image target/java-runtime \
+--icon ${ICON_PATH} \
+--app-version ${APP_VERSION} \
+--vendor "Gluon" \
+--copyright "Copyright © 2021 Gluon" \
+--type app-image
+"$@"
```
Comment (abhinayagarwal) [inline content/javafx/packaging/_index.md:86]: This is not required.

File: content/javafx/packaging/_index.md (hunk 1 above)
Comment (johanvos) [reply content/javafx/packaging/_index.md:66]: Can you make a suggestion for this?

**Review by abhinayagarwal — APPROVED** [review]

**Pull request merged**